### DISCUSSION TITLE
Any idea for load and read a local indexdb using the component?

### DISCUSSION THREAD

**Discussion by yhojann-cl** [dumbmatter/fakeIndexedDB#100] [General]
Have a [Fluent Reader](https://flathub.org/apps/me.hyliu.fluentreader) app and need extract all favourites news in a single list for publish in other website from a CLI nodejs project without a web browser. I try read the documentation but i don't know load a external database for work with it. Have a single example?

My app index path is:

```javascript
const os = require('os');

const idbPath =`${os.homedir()}/.var/app/me.hyliu.fluentreader/config/fluent-reader/IndexedDB/filefile__0.indexeddb.leveldb`;
```

**Comment by dumbmatter** [comment]
No, see #97
